torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6444,6 +6470,22 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_gradgrad=True,
            supports_out=False,
            autodiff_nonfusible_nodes=["aten::leaky_relu"]),
+    OpInfo('nn.functional.linear',
```

Comment:
I would add a comment before these entries for why they're separate. Typically we only separate an op into multiple entries if it has different metadata (like different dtypes) for different inputs. I imagine there's a reason this wants to treat linear with and without bias as separate operations?

Reply:
I don't have a good reason for why these should be separate, I was acting on reflex. I combined the two cases and now am modeling this as a single op; if someone else comes along with a use case to separate them in the future we could do that then.